TramsDataApi/Gateways/AcademyConversionProjectGateway.cs

```diff
@@ -76,6 +76,11 @@ public async Task<PagedResult<AcademyConversionProject>> SearchProjects(int page
             {
                 var lowerDeliveryOfficers = deliveryOfficers.Select(officer => officer.ToLower());
                 academyConversionProjects = academyConversionProjects.Where(acp => lowerDeliveryOfficers.Contains(acp.AssignedUserFullName.ToLower()));
+                if (lowerDeliveryOfficers.Contains("Not assigned"))
+                {
+                    var notAssignedProjects = academyConversionProjects.Where(acp => acp.AssignedUserFullName == "" && acp.AssignedUserFullName == null);
```

Comment:
You can shorten this to string.IsNullOrEmpty()

Reply:
Done